templates/php.properties.json

```diff
@@ -1,3 +1,26 @@
 {
-  "iconName": "php"
+  "iconName": "php",
+  "parameters": [
+        {
+            "name": "azureRmConnection",
+            "type": "endpoint:azureRm",
+            "required": "true"
+        },
+        {
+            "name": "webAppName",
```

Comment:
Can we make the dropdown editable?
Few customers may want to add variables here